cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDClaim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+
+@Component
+public class ClaimMapper implements Mapper<CCDClaim, ClaimData> {
+
+    private final PersonalInjuryMapper personalInjuryMapper;
+    private final HousingDisrepairMapper housingDisrepairMapper;
+    private final StatementOfTruthMapper statementOfTruthMapper;
+    private final PartyMapper partyMapper;
+
+    @Autowired
+    public ClaimMapper(PersonalInjuryMapper personalInjuryMapper, HousingDisrepairMapper housingDisrepairMapper,
+                       StatementOfTruthMapper statementOfTruthMapper, PartyMapper partyMapper) {
+        this.personalInjuryMapper = personalInjuryMapper;
+        this.housingDisrepairMapper = housingDisrepairMapper;
+        this.statementOfTruthMapper = statementOfTruthMapper;
+        this.partyMapper = partyMapper;
+    }
+
+    @Override
+    public CCDClaim to(ClaimData claimData) {
+        final CCDClaim.CCDClaimBuilder builder = CCDClaim.builder();
+        claimData.getFeeCode().ifPresent(builder::feeCode);
+        claimData.getFeeAccountNumber().ifPresent(builder::feeAccountNumber);
+        claimData.getExternalReferenceNumber().ifPresent(builder::externalReferenceNumber);
+        claimData.getPreferredCourt().ifPresent(builder::preferredCourt);
+
+        claimData.getStatementOfTruth()
+            .ifPresent(statementOfTruth -> builder.statementOfTruth(statementOfTruthMapper.to(statementOfTruth)));
+
+        claimData.getPersonalInjury()
+            .ifPresent(personalInjury -> builder.personalInjury(personalInjuryMapper.to(personalInjury)));
+
+        claimData.getHousingDisrepair()
+            .ifPresent(housingDisrepair -> builder.housingDisrepair(housingDisrepairMapper.to(housingDisrepair)));
+
+        claimData.getClaimants().forEach(partyMapper::to);
+
+        return builder
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1135)